helpers/deps.py

```
@@ -149,40 +180,56 @@ def _find_available_powershell():
 
 def _add_to_path_env(bin_folder):
     powershell = _find_available_powershell()
-    command = f'{powershell} -Command "[Environment]::SetEnvironmentVariable(\'Path\', [Environment]::GetEnvironmentVariable(\'Path\', \'User\') + \';{bin_folder}\', \'User\')"'
+    command = (
+        f"{powershell} -Command \""
+        f"[Environment]::SetEnvironmentVariable('Path', "
+        f"[Environment]::GetEnvironmentVariable('Path', 'User') + ';{bin_folder}', 'User')\""
+    )
     subprocess.run(command, shell=True, check=True)
```

Ruff (0.16.2)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)